### DISCUSSION TITLE
using HTML-Webpack-plugin to use multiple HTML files, but one of them cannot execute JS script?

### DISCUSSION THREAD

**Discussion by ClemGG** [webpack/webpack#18872] [Q&A]
Hello, this is probably is a newbie question as I'm sure the answer is very blatant, but I've been at it for hours and I can't figure it out.

I'm using webpack to compile a project using the BabylonJS framework for rendering 3D simulations. The main files are here: 
#https://github.com/ClemGG/filedump/tree/main/babylonsxd

In the webpack.config.js file, I create multiple htmlPlugins that I then pass to the plugins options of the file. Running ` npm build run` successfully produces the bundle files and the copies of the html files to the dist folder.

Opening the index file in localhost, things seem to run just fine, but when opening the editor page by pressing enter, the script referenced in editor.html (dist/editor.bundles.js) doesn't seem to get called. The terminal shows no compilation errors, and neither the console nor the network windows throw any exception. I've checked if the issue came from the HTML side, but everything seems to be in order. It's like the file doesn't even recognize the <script> tag in both the body and the head of the editor.html file, while it does in the index.html file.

I have no idea what could be causing this. Any help would be welcome. Thank you for your time.

**Comment by alexander-akait** [comment]
Unfortunately your repository is private and I can't see it, maybe you forgot to inject `runtime.js`, can you show configuration and `dist` files?

**Reply by ClemGG** [reply]
Hi, sorry, I wasn't paying attention to the visibility of the repo. I marked it as public so you can see the files.

**Reply by alexander-akait** [reply]
Sorry, I can't help, because you don't provide full project, I can't find `'./src/index.ts'` and `'./src/editor.ts'`, 

but your `script` is in the wrong place, you can't have it there, put it in the body

```
</body>

  <script>

    /**
    * Affiche l'écran de chargement au lancement de la fenêtre
    */
    function showLoadingScreen()
    {
        if (document.all)
        {
            document.all["loading"].style.visibility="visible";
        } 
        else if (document.getElementById)
        {
            node = document.getElementById("loading").style.visibility='hidden';
        }
    }

  </script>
  
</html>
```
